File: Java/program45.java
///////////////////////////////////////////////////////
//
//  File name :     program45.java
//  Descreption :   Separate Digit and Display from Number
//  Author :        Vaibhav
//  Date :          20/05/2025
//
///////////////////////////////////////////////////////

import java.util.Scanner;

class program45
{
    public static void main(String A[])
    {
        int iNo = 751;
        int iDigit = 0; 

        System.out.println("-----------------------------------------------------------------------------------------------");
        System.out.println("Number Received is "+iNo);
        System.out.println("-----------------------------------------------------------------------------------------------");
        
        iDigit = iNo % 10;
        System.out.println("Digit is : "+iDigit);       // 1
        iNo = iNo / 10;
        System.out.println("Updated iNo is "+iNo);      //75

        iDigit = iNo % 10;
        System.out.println("Digit is : "+iDigit);       //5
        iNo = iNo / 10;
        System.out.println("Updated iNo is "+iNo);      //7

        iDigit = iNo % 10;
        System.out.println("Digit is : "+iDigit);       //7
        iNo = iNo / 10;
        System.out.println("Updated iNo is "+iNo);      //0
        
    }
}

File: program45.c
///////////////////////////////////////////////////////
//
//  File name :     program45.c
//  Descreption :   Separate Digit and Display from Number
//  Author :        Vaibhav
//  Date :          12/05/2025
//
///////////////////////////////////////////////////////


#include<stdio.h>

int main()
{

    int iNo = 751;
    int iDigit = 0;
    
    printf("-----------------------------------------------------------------------------------------------\n");
    printf("Number Received is %d\n",iNo);
    printf("-----------------------------------------------------------------------------------------------\n");

    iDigit = iNo % 10;
    printf("Digit is : %d\n",iDigit);      //1
    iNo = iNo / 10;
    printf("updated iNo : %d\n",iNo);       //75


    iDigit = iNo % 10;
    printf("Digit is : %d\n",iDigit);      //5
    iNo = iNo / 10;
    printf("updated iNo : %d\n",iNo);       //7

    iDigit = iNo % 10;
    printf("Digit is : %d\n",iDigit);      //7
    iNo = iNo / 10;
    printf("updated iNo : %d\n",iNo);       //0

    return 0;
}